test/Confluent.Kafka.IntegrationTests/Tests/Producer_Poll.cs

```diff
@@ -47,7 +47,8 @@ public void Producer_Poll(string bootstrapServers)
                 // call to Poll (or if the poll method hasn't been called, over
                 // the lifetime of the producer).
                 Assert.True(producer.Poll(TimeSpan.FromMilliseconds(500)) >= 1);
-                Assert.True(sw.ElapsedMilliseconds < 1);
+                var elapsed = sw.ElapsedMilliseconds;
+                Assert.True(elapsed < 2);
```

Comment:
This suddenly stopped working today. a lot has changed about my environment since yesterday (new distro, new .net install), so there's a good chance the failure is related to that. But please have a quick think about whether anything in librdkafka change that could make poll take longer.